src/DataModel/IResource.ts

```diff
@@ -0,0 +1,5 @@
+export interface IResource
+{
+    "@id": string;
+    "@type"?: Array<string>
```

Comment:
Yes - I don't like _@id_ either. The issue is that it's not something I can do with i.e. JSON-LD framing.  Initially I had here an _iri_ property.